internal/periodic_checker/server.go

```diff
@@ -33,25 +33,38 @@ func New(
 	}
 }
 
+// TODO: expose metrics
 func (s *Server) Run() error {
 	provider := queues.NewAmqpProvider(s.cfg.Queue.AmqpURL, s.log)
 	defer provider.Stop()
-	repoTester := tasks.NewRepoTester(s.log, s.store)
-	repoTesterThread := thread.New(
-		s.log.WithField("pkg", "repository-tester"), "Repository tester", 2*time.Minute, repoTester.TestRepositories)
-	repoTesterThread.Start()
-	defer repoTesterThread.Stop()
 
 	publisher, err := tasks.TaskQueuePublisher(provider)
 	if err != nil {
 		return err
 	}
 	callbackManager := tasks.NewCallbackManager(publisher, s.log)
+
+	// repository tester
+	repoTester := tasks.NewRepoTester(s.log, s.store)
+	repoTesterThread := thread.New(
+		s.log.WithField("pkg", "repository-tester"), "Repository tester", 2*time.Minute, repoTester.TestRepositories)
+	repoTesterThread.Start()
+	defer repoTesterThread.Stop()
+
+	// resource sync
 	resourceSync := tasks.NewResourceSync(callbackManager, s.store, s.log)
 	resourceSyncThread := thread.New(
 		s.log.WithField("pkg", "resourcesync"), "ResourceSync", 2*time.Minute, resourceSync.Poll)
 	resourceSyncThread.Start()
 	defer resourceSyncThread.Stop()
+
+	// device liveness
+	deviceLiveness := tasks.NewDeviceLiveness(s.log, s.store)
```

Comment:
Should this be "Disconnected" or "Offline" instead of "Liveness"? "Liveness" would be a bit misleading, because the device may be simply without network coverage but fully operational.

Reply:
Updated to Disconnected to make progress